### PR TITLE
Fix build validation warning and suggestions in topics - unique titles for US1889327

### PR DESCRIPTION
**Pull request by kirupap**
Fix build validation warning/suggestions for US1889327

Files Changed: 2
1. How to Create an Unbound Windows Forms DataGridView Control
2. Walkthrough to Populate an Unbound Windows Forms DataGridView Control

Fixed unique titles and added description to each topic.

### CODE REVIEW

**Comment by kirupap** [comment]
Fix build validation warning/suggestions for US1889327

Files Changed: 2 (duplicate tiles)+1(duplicate heading)

1. How to Create an Unbound Windows Forms DataGridView Control

2. Walkthrough to Populate an Unbound Windows Forms DataGridView Control

3. How-to-alter-the-typography-of-text

Fixed unique titles and added description to each topic to metadata
Fixed unique level 2 headings and added description to metadata

**Review by adegeo — APPROVED** [review]
Thanks!

File: dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
-title: Create an Unbound DataGridView Control
+title: How to Create an Unbound Windows Forms DataGridView Control
+
+description: Populate an unbound Windows Forms DataGridView Control programmatically and display a small amount of data in a table format without binding it to a data source
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md:4]: @kirupap Right after I merged I noticed this blank line. Generally there aren't any blank lines between metadata items. Can you fix these up?

File: dotnet-desktop-guide/framework/winforms/controls/walkthrough-creating-an-unbound-windows-forms-datagridview-control.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
-title: Create an Unbound DataGridView Control
+title: Walkthrough to Creating an Unbound Windows Forms DataGridView Control
+
+description: You can create an Unbound Windows Forms DataGridView Control and display a small amount of data without binding it to a data source
+
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/walkthrough-creating-an-unbound-windows-forms-datagridview-control.md:5]: Same here

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ---
-title: "How-to: Alter the Typography of Text"
+title: How to Alter the Typography of Text
+
+description: To alter the telegraphy of text, set the telegraphy attribute. You can set the typographic property of text programmatically. You can see how altered and default telegraphy properties of text render on screen each with an example.
+
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md:5]: And here

**Pull request merged**